src/Stakeholders/components/Agencies/NotificationForm/index.js

```diff
@@ -107,7 +112,7 @@ class NotificationForm extends Component {
           })(
             <SearchableSelectInput
               placeholder="Enter notification recipients"
-              onSearch={getStakeholders}
+              onSearch={get}
```

Comment:
should crosscheck this one

Reply:
I have just reused the logic from here, https://github.com/CodeTanzania/emis-web/blob/92f20b255727f5381b6037a81f2b484484ba0e6d/src/Stakeholders/components/Contacts/NotificationForm/index.js#L112 . I was epecting it to search contacts

Reply:
This is a bug and thanks for pointing it out.. 
it is supposed to be like

```js
import { httpActions } from '@codetanzania/emis-api-client';

/* constants */
const { getAgencies } = httpActions;
```
 then use it there

Reply:
okay, still I feel like it doesn't much make sense  to share agencies to an agency.  what if we make it share agencies to a stakeholder

Reply:
Sure you can just  use ```js const { getFocalPeople } = httpActions;``` instead of agencies. I thought the form was used for notification and not sharing

Reply:
Also what do you think about the notify feature in agencies, I feelike it is not applicable, I feel like share, export and archive actions make more sense, should we remove notify agency feature ?

Reply:
what if we wan't to notify people in a specific agency?, is this use case relevant?

Reply:
I think we have filters in focal persons, you can filter by agency then apply bulk action(the checkbox), then notify. I feel like the focal persons component can best handle that..

Reply:
Mark it as a discussion point but for now you can handle share action first and comment out notify

Reply:
Okay, let  me  do that